web/src/utils/doNotTrack.ts

```diff
@@ -0,0 +1,8 @@
+// Vimeo supports do not track
```

Comment:
I think this is self explanatory, isn't it? :see_no_evil:

Reply:
Does it make sense to move this to the api-client as this is probably the same logic for native?

Reply:
In general this makes sense, but the problem is we can not import function in the string template. Or lets say import functions and pass them parameters. I tried this several time with the `t()` to pass sources to the translations